src/node/Launcher.ts

```diff
@@ -587,6 +595,35 @@ class FirefoxLauncher implements ProductLauncher {
   }
 }
 
+function executablePathForChannel(channel: ChromeReleaseChannel): string {
+  const platform = os.platform();
```

Comment:
WDYT about checking if the path that we assumed here exists and if not throw an error specific to this situation?

Reply:
The check should be added here, showing an error like 'Browser is not installed'.